setup.py

```diff
@@ -23,7 +23,7 @@
     install_requires=[
         "six",
         "tqdm~=4.30.0",
-        "torch>=1.2,<1.4",  # Originally torch==1.2.0, changed for compatibility with powerai-1.7.0 (pytorch-1.3.1)
+        "torch>=1.2,<1.5",  # Originally torch==1.2.0, changed for compatibility with powerai-1.7.0 (pytorch-1.3.1)
```

Comment:
```suggestion
        "torch>=1.2,<1.5",
```
Let's remove this unnecessary comment at the same time